### ISSUE TITLE
treeees

### ISSUE THREAD

**Issue by ajschumacher** [ajschumacher/ajschumacher.github.io#290]
LightGBM
 * smart categorical splits
 * leaf-wise tree growth
 * focuses on high-gradient training examples
 * uses exclusive feature bundling

**Comment by ajschumacher** [comment]
CatBoost

 * oblivious trees
 * category interactions
 * permutation for category score transform and boosting

**Comment by ajschumacher** [comment]
XGBoost

* handles sparsity
* weighted quantile sketch
* regularization
* out-of-core
